activiti-spring-identity/src/main/java/org/activiti/spring/identity/ActivitiUserGroupManagerImpl.java

```diff
@@ -12,9 +12,11 @@
 public class ActivitiUserGroupManagerImpl implements UserGroupManager {
 
     private final UserDetailsService userDetailsService;
```

Comment:
@pancudaniel7 here is interesting.. you can keep using the UserDetailsService in the constructor, but send an ExtendedInMemoryUserDetailsManager instance. Then inside the getGroups and getUsers you can cast to the specific type if the UserDetailsService is instance of ExtendedInMemoryUserDetailsManager, but you never need both